src/server/inversify.config.ts

```diff
@@ -161,4 +143,6 @@ export default () => {
   }
 
   bindIfUnbound(DependencyIds.s3, S3ServerSide)
+
+  addRedirectBindings(container)
```

Comment:
i'm still trying to understand the value of this abstraction, given that all the other bindings are performed inline above, and other developers can tell at a glance what's happening. is it supposed to be the start of introducing a loader pattern? but then, the implementation is specific to the redirect module.

it's only partially abstracting away DI, which is an edge case. can we leave it out as a separate PR until we're ready to tackle the entire thing?